ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -1,6 +1,8 @@
 from styx_msgs.msg import TrafficLight
```

Comment:
Hm...What is the garbage about? I only get a few lines about TF not being configured to use certain operations available on my machine, but that's normal. I don't want to disable logging completely in case there are actual errors.